### PR TITLE
Adds release notes for 2.4.16

### PR DESCRIPTION
**Pull request by Gowiem**

### CODE REVIEW

File: _posts/releases/2015-06-26-ios-release-2-4-16.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: iOS 2.4.16 Release Notes
+author: "Artisan"
+category: release-notes
+description: "What's new in Artisan's Mobile Experience Management platform."
+---
+# iOS 2.4.16 Release Notes
+
+Released June 26, 2015
+
+* [FIXED] Changes to make iOS SDK more resilient in the case of receiving failed HTTP requests from Artisan servers.
```
Comment (atroutt) [inline _posts/releases/2015-06-26-ios-release-2-4-16.md]: the change in this release is really about configuration and playlist downloads if we want to be specific. It doesn't address resiliance for *all* requests.

File: _posts/ios/2014-05-09-appledocs.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ description: "Links to Appledocs for Artisan iOS"
 
 For more in-depth documentation of our public API please see the Appledocs for the version of the iOS SDK that you are using.
 
+* <a target="_blank" href="/ios/appledoc/2_4_16">2.4.16</a>
+* <a target="_blank" href="/ios/appledoc/2_4_15">2.4.15</a>
```
Comment (atroutt) [inline _posts/ios/2014-05-09-appledocs.md]: did this file just not get committed for 2.4.15?

**Comment by atroutt** [comment]
I think this looks good :+1:

File: _posts/ios/2014-05-09-appledocs.md (hunk 2 above)
Comment (Gowiem) [inline _posts/ios/2014-05-09-appledocs.md]: Nope.

**Pull request merged**